<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SaveFormAsTemplate</class>
 <widget class="QDialog" name="SaveFormAsTemplate" >
  <property name="objectName" >
   <string notr="true" >SaveFormAsTemplate</string>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Save Form As Template...</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="objectName" >
    <string notr="true" />
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="objectName" >
      <string notr="true" />
     </property>
     <property name="margin" >
      <number>1</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="templateNameEdit" >
       <property name="objectName" >
        <string notr="true" >templateNameEdit</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>60</x>
         <y>3</y>
         <width>376</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="categoryCombo" >
       <property name="objectName" >
        <string notr="true" >categoryCombo</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>60</x>
         <y>31</y>
         <width>376</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="objectName" >
        <string notr="true" >label_2</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>1</x>
         <y>31</y>
         <width>53</width>
         <height>24</height>
        </rect>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text" >
        <string>&amp;Category:</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::AutoText</enum>
       </property>
       <property name="buddy" >
        <cstring>categoryCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="objectName" >
        <string notr="true" >label</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>1</x>
         <y>3</y>
         <width>53</width>
         <height>23</height>
        </rect>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text" >
        <string>&amp;Name:</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::AutoText</enum>
       </property>
       <property name="buddy" >
        <cstring>templateNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QToolButton" name="toolButton" >
       <property name="objectName" >
        <string notr="true" >toolButton</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>444</x>
         <y>31</y>
         <width>15</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="objectName" >
      <string notr="true" >verticalSpacer_2</string>
     </property>
     <property name="geometry" >
      <rect>
       <x>9</x>
       <y>74</y>
       <width>463</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="horizontalLine" >
     <property name="objectName" >
      <string notr="true" >horizontalLine</string>
     </property>
     <property name="geometry" >
      <rect>
       <x>9</x>
       <y>102</y>
       <width>463</width>
       <height>3</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="objectName" >
      <string notr="true" />
     </property>
     <property name="margin" >
      <number>1</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="objectName" >
        <string notr="true" >horizontalSpacer</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>1</x>
         <y>1</y>
         <width>297</width>
         <height>29</height>
        </rect>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="objectName" >
        <string notr="true" >okButton</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>304</x>
         <y>1</y>
         <width>77</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="objectName" >
        <string notr="true" >cancelButton</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>387</x>
         <y>2</y>
         <width>75</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
</ui>
